### PR TITLE
Fix repolist icons

### LINKED ISSUE #12226: Octicons are missing for dashboard repolist
- Gitea version (or commit ref): bac57ab5900672357fb0a36f9433e516088bd21b
- Git version: `gitea:latest`
- Operating system: `gitea:latest`
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant

## Description

https://github.com/go-gitea/gitea/pull/12157 introduced direct SVG rendering however no appropriate changes were made to dashboard repolist https://github.com/go-gitea/gitea/blob/bac57ab5900672357fb0a36f9433e516088bd21b/templates/user/dashboard/repolist.tmpl#L64

### PR DESCRIPTION
**Pull request by silverwind**
We should really extract this file to its own .vue template as having multiple template languages in one file is majorly confusing.

Fixes: https://github.com/go-gitea/gitea/issues/12226

### CODE REVIEW

**Comment by silverwind** [comment]
Something for later (after extracting to standalone vue component):

https://github.com/visualfanatic/vue-svg-loader

**Review by zeripath — APPROVED** [review]

File: web_src/js/index.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ import initTableSort from './features/tablesort.js';
 import ActivityTopAuthors from './components/ActivityTopAuthors.vue';
 import {initNotificationsTable, initNotificationCount} from './features/notification.js';
 import {createCodeEditor} from './features/codeeditor.js';
+import {svg} from './svg.js';
```
Comment (zeripath) [inline web_src/js/index.js]: Are you sure this is necessary?

Comment (silverwind) [reply web_src/js/index.js]: No, it's a leftover from a previous attempt.

Comment (silverwind) [inline web_src/js/index.js]:
```suggestion
```

**Review by lafriks — APPROVED** [review]

**Pull request merged**